client/src/services/batchList/actions.tsx

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
+import { IAction } from 'types';
+import { RIVER_URL } from '../../constants';
+
+const listBatch = (): IAction => {
+  return async (dispatch: any) => {
+    try {
+      const { data } = await axios.get(`${RIVER_URL}/batch`);
+      dispatch({
+        type: 'LIST_BATCH_SUCCESS',
+        payload: data
+      });
```

Comment:
@BPierrick, is it a good way to handle redux and axios calls?

Reply:
One proper modern way would have been to use RTK: first a slice which include the reducer and its actions and here an async thunk (https://redux-toolkit.js.org/api/createAsyncThunk) handling the request and its side effects. Usually  we also need to create a wrapper around axios instead of calling it directly and use interceptors if needed for each API. 

But since we don't use redux toolkit in this project and there's no pre-established architecture of the redux store for handling side effects I chose to not do any refactoring especially since it will soon become legacy code.